### PR TITLE
Load all parameters into data, even if the question never comes up. 

### PR DESCRIPTION
**Pull request by NilesDev**
Otherwise things entered with -a or -p options might not be included in the final product. 

Also release of the code execution update!

### CODE REVIEW

File: jobbergate/cli.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ def ask_questions(fields, answerfile, use_defaults=False):
     # Check if questions has already been answered
     for question in flatten(questions):
         if question.name in answerfile:
-            retval.update({question.name: answerfile[question.name]})
+            pass
         elif use_defaults and question.default is not None:
```
Comment (HeMan) [inline jobbergate/cli.py]: Can't the `if question.name in answerfile` be removed and the `elif` be changed to `if`?

**Comment by HeMan** [comment]
I think version bumping is best done in master, along with tagging. The tagging makes it deploy on pypi as well.

File: jobbergate/cli.py (hunk 1 above)
Comment (NilesDev) [reply jobbergate/cli.py]: Not outright, pre-supplied answers take precedence.
If it is already answered, both using defaults and actually asking the question should not happen.

We could do something like :
```
if not already answered and [-f flag]:
    use default
elif not already answered:
    ask the question
```

But I think the current one is slightly clearer in what's going on.

**Comment by NilesDev** [comment]
Oh, my bad, I thought pypi deployment only happened when a tag shows up in the _master_ branch.

**Comment by HeMan** [comment]
Yes, it's when it's in the master branch. That's why I think all version bumping should be done there.

**Comment by HeMan** [comment]
Or at least "production" versioning, ie -dev-versions could be in a branch.

**Comment by NilesDev** [comment]
Ok, I'll merge this one soon but in the future we'll do the bumping and tagging directly in the master branch.

**Comment by HeMan** [comment]
I could add the tag in github after your merge so that it deploys.

**Pull request merged**